service/src/integration-test/resources/json/expectedResult.json

```diff
@@ -20,10 +20,10 @@
         },
         "destination": [
           {
-            "endpoint": "urn:nhs-uk:addressing:ods:EM396"
+            "endpoint": "urn:nhs-uk:addressing:ods:EM396:DOSServiceID:26428"
```

Comment:
I think only expectedResult2 should expect this endpoint value (only ITK_Report_request_2.xml has both ODS Code and DOS Service ID).